Components/Queues/Drivers/StompQueueDriver.php

```diff
@@ -93,13 +93,34 @@ public function getStompVersion()
     }
 
     /**
-     * @param string $stompVersion
+     * Set the version to Stomp driver
+     *
+     * @param string $version
      */
-    public function setStompVersion($stompVersion)
+    public function setStompVersion(string $version = self::STOMP_VERSION)
     {
-        $this->stompVersion = $stompVersion;
+        $this->stompVersion = $version;
     }
 
+    /**
+     * Set the timeout to Stomp driver
+     *
+     * @param int $timeout
+     */
+    public function setTimeout(int $timeout = 3)
+    {
+        $this->timeout = $timeout;
+    }
+
+    /**
+     * Set if the driver will work in synchronous or asynchronous mode
+     *
+     * @param bool $sync
+     */
+    public function setSync(bool $sync = true)
+    {
+        $this->sync = $sync;
```

Comment:
The property $sync is not defined in the StompQueueDriver class

Reply:
sorry, it is defined now.